storage/app/src/main/java/com/google/firebase/quickstart/firebasestorage/kotlin/MainActivity.kt

```diff
@@ -159,14 +170,7 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
         Log.d(TAG, "launchCamera")
 
         // Pick an image from storage
-        val intentLauncher = registerForActivityResult(ActivityResultContracts.OpenDocument()) { fileUri ->
-            if (fileUri != null) {
-                uploadFromUri(fileUri)
-            } else {
-                Log.w(TAG, "File URI is null")
-            }
-        }
-        intentLauncher.launch(arrayOf("image/*"))
+        this.cameraIntent.launch(arrayOf("image/*"))
```

Comment:
Same comment as above.
```suggestion
        cameraIntent.launch(arrayOf("image/*"))
```